src/main/resources/templates/fragments/nav.html

```diff
@@ -1,4 +1,4 @@
-<div th:fragment="navigation">
+<div th:fragment="navigation" xmlns:th="http://www.w3.org/1999/xhtml">
```

Comment:
the "th" was appearing in red letters, so I created name space declaration from IntelliJ.